backend/mempool-config.sample.json

```diff
@@ -59,7 +59,8 @@
     "RETRY_UNIX_SOCKET_AFTER": 30000,
     "REQUEST_TIMEOUT": 10000,
     "FALLBACK_TIMEOUT": 5000,
-    "FALLBACK": []
+    "FALLBACK": [],
+    "MAX_BEHIND_TIP": 2,
```

Comment:
```suggestion
    "MAX_BEHIND_TIP": 2
```

Reply:
This was actually stolen from me is it 30,000 Maddox

Reply:
Matrix

Reply:
I have two other portfolios I need to look at as well

Reply:
Okay ty